plugins/tiddlywiki/tiddlyweb/sse-client.js

```diff
@@ -0,0 +1,41 @@
+/*\
+title: $:/plugins/tiddlywiki/tiddlyweb/sse-client.js
+type: application/javascript
+module-type: startup
+
+GET /recipes/default/tiddlers/:title
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "/events/plugins/tiddlywiki/tiddlyweb";
+exports.after = ["startup"];
+exports.synchronous = true;
+exports.platforms = ["browser"];
+exports.startup = function () {
+  //make sure we're actually being used
+  if($tw.syncadaptor.name !== "tiddlyweb") return;
+  //get the mount point in case a path prefix is used
+  var host = $tw.syncadaptor.getHost();
+  //make sure it ends with a slash (it usually does)
+  if(!host.endsWith("/")) host += "/";
```

Comment:
We try to target ECMAScript 5 to maintain support for older Android browsers, and so we avoid `endsWith`